src/hooks/useKyberSwapConfig.ts

```diff
@@ -65,7 +65,8 @@ const parseGlobalResponse = (
   chainId: ChainId,
 ): KyberswapGlobalConfig => {
   const data = responseData?.data?.config
-  const aggregatorDomain = data?.aggregator ?? DEFAULT_AGGREGATOR_API
+  const aggregatorDomain =
+    chainId === ChainId.LINEA_TESTNET ? DEFAULT_AGGREGATOR_API : data?.aggregator ?? DEFAULT_AGGREGATOR_API
```

Comment:
Why would we need this

Reply:
Meta prod needs authorization. then for linea testnet, just directly call aggregator is better

Reply:
We might need to make `aggregator` config chain based

Reply:
DEFAULT_AGGREGATOR_API is meta

rename DEFAULT_AGGREGATOR_API => META_AGGREGATOR_API to prevent confused

Reply:
will revert now. this is only for testing